torch/testing/_internal/common_cuda.py

```diff
@@ -32,3 +34,52 @@ def initialize_cuda_context_rng():
         for i in range(torch.cuda.device_count()):
             torch.randn(1, device="cuda:{}".format(i))
         __cuda_ctx_rng_initialized = True
+
+
+def tf32_is_not_fp32():
+    if not torch.cuda.is_available():
+        return False
+    if torch.cuda.get_device_properties(torch.cuda.current_device()).major < 8:
+        return False
+    if int(torch.version.cuda.split('.')[0]) < 11:
+        return False
+    return True
+
+
```

Comment:
Better add a comment and an example here

Reply:
added